data/coreteam_members.yml

```diff
@@ -107,4 +107,9 @@ members:
     description: |
       a computer science undergrad who's having fun learning new things.
       He enjoys reading, football, and foods he hasn't tried before. Reach him at <a href="mailto:jivesh@nushackers.org">jivesh@nushackers.org</a>
+  
+  - name: Azeem
+    description: |
+      a computer science undergrad who loves tinkering and building things
```

Comment:
Needs a full stop.

Reply:
undergrad is slang. Please use "undergraduate" instead if not it will break our entire build pipeline and cause production outage.

Reply:
Fixed